examples/design/EU-DEMO/EUDEMO_reactor.py

```diff
@@ -154,11 +154,11 @@
     "tk_tf_ins": 0.08,
     "tk_bb_ib": 0.755,
     "tk_bb_ob": 1.275,
-    "g_tf_pf": 0.05,
+    # "g_tf_pf": 0.05,
     "C_Ejima": 0.3,
     "eta_nb": 0.4,
-    "LPangle": -15,
-    "w_g_support": 1.5,
+    # "LPangle": -15,
+    # "w_g_support": 1.5,
```

Comment:
Can these be removed? Feels like we're adding unnecessary tech debt in needing to remember to uncomment/remove these later